### ISSUE TITLE
Error: '~/AppData/Local/Temp/RtmpWgW3LK/names.csv' does not exist.

### ISSUE THREAD

**Issue by stijndelaat** [nirmalpatel/fuzzymineR#2]
Great job on making an R package for fuzzy mining! I am currently trying to use it on my own data and have a question.
When creating the metrics with mine_fuzzy_model() R gives the following error:

`Error: '~/AppData/Local/Temp/RtmpWgW3LK/names.csv' does not exist.`

Which is true, but I don't know how to solve this error. Is there an easy fix or am I doing something completely wrong? I would appreciate your help!

**Comment by nirmalpatel** [comment]
Thanks for reporting this. We have seen this issue in different situations on and off, but it is a bug that is hard to reproduce.

Can you try removing special chars from the names of the events and give it a try?

**Comment by stijndelaat** [comment]
Thank you for the quick response. Unfortunately this did not solve the problem. There were special characters in the names, but I have removed them:

```
log2 <- df3 %>%
  create_eventlog(
    case_id = "case",
    activity_id = "RegPhase",
    timestamp = "Time",
    resource_id = "Subject"
  )
```

**Comment by nirmalpatel** [comment]
Can you tell me what OS you are using and more details about the OS like version, etc?

Also, can I know your R version that comes out when you use the "version" command in the R console?

Thanks!

**Comment by stijndelaat** [comment]
I have windows 10 Home edition 64 bit.
```
> version()
platform       x86_64-w64-mingw32          
arch           x86_64                      
os             mingw32                     
system         x86_64, mingw32             
status                                     
major          4                           
minor          0.2                         
year           2020                        
month          06                          
day            22                          
svn rev        78730                       
language       R                           
version.string R version 4.0.2 (2020-06-22)
nickname       Taking Off Again 
```

**Comment by nirmalpatel** [comment]
So I looked through this file https://github.com/nirmalpatel/fuzzymineR/blob/master/R/mine_fuzzy_model.R

I'm trying to figure out whether this is a Windows-specific issue or not... You can see that all of the file paths are constructed using file.path() so that looks fine. Do you have Java installed? We invoke a JAR file on line 155...

**Comment by stijndelaat** [comment]
I have Java installed (V8. update 271).

Guess this is the point where my knowledge in R stops :). I have no idea what a JAR file is or how to properly source a function from a package and how to use the browser function the way you mean. 

I have tried the following:
`test<-fuzzymineR:::mine_fuzzy_model`
Then opened "test" via the global environment and ran the code line by line. It threw an error when I hit line 28:
`cli_template <- read_lines(cli_template_fpath)`
`Error in read_lines(cli_template_fpath) : 
  could not find function "read_lines"`

Am I right in thinking that this is not the big error we were looking for, but it is due to the way I tried to look for the error?
